lib/Conversion/TritonGPUToLLVM/HistogramOpToLLVM.cpp

```diff
@@ -36,19 +35,23 @@ computeWarpLevelHistogram(Location loc, RankedTensorType srcType,
     SmallVector<Value> ballotBits;
     for (int j = 0; j < numBits; ++j) {
       Value bitSet = and_(value, i32_val(1 << j));
-      Value threadMask = i32_val(-1);
-      Value bit = rewriter.create<NVVM::VoteBallotOp>(loc, i32_ty, threadMask,
-                                                      icmp_ne(bitSet, zero));
+      Value threadMask = int_val(numThreadPerWarp, -1);
+      Value cmp = icmp_ne(bitSet, zero);
+      Value bit = targetInfo.callBallotOp(
+          rewriter, loc, int_ty(numThreadPerWarp), threadMask, cmp);
       ballotBits.push_back(bit);
     }
-    Value fullMask = i32_val(0xFFFFFFFF);
+    uint64_t fullMaskValue =
```

Comment:
@ThomasRaoux is there a more generic way to do this ? Seems at least I should assert that numThreadPerWarp is either 32 or 64

Reply:
I think `0xFFFFFFFFFFFFFFFF` should work for both case since we are creating a value of 32 or 64 bits in the following line.
asserting that `numThreadPerWarp` is 32 or 64 makes sense